--- src/compositor/compositor-xrender.c.orig	2009-04-25 18:43:57.000000000 -0300
+++ src/compositor/compositor-xrender.c	2009-04-25 18:44:43.000000000 -0300
@@ -1451,7 +1451,7 @@
                                             NULL);
 #else
   /* Limit it to 50fps */
-  compositor->repaint_id = g_timeout_add_full (G_PRIORITY_HIGH, 20,
+  compositor->repaint_id = g_timeout_add_full (G_PRIORITY_HIGH, 13,
                                                compositor_idle_cb, compositor,
                                                NULL);
 #endif
